### PR TITLE
Fix some linting and mispelling issues

### PR DESCRIPTION
**Pull request by fredbi**

### CODE REVIEW

File: scan/classifier_test.go [hunk 1]
```diff
@@ -131,10 +131,10 @@ func TestClassifierInclude(t *testing.T) {
 	prog := classificationProg
 	classifier := &programClassifier{
 		Includes: packageFilters([]packageFilter{
-			packageFilter{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification"},
-			packageFilter{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification/transitive/mods"},
-			packageFilter{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification/operations"},
-			packageFilter{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification/operations_annotation"},
+			{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification"},
```
Comment (golangci) [inline scan/classifier_test.go]: github.com/go-swagger/go-swagger/scan.packageFilter composite literal uses unkeyed fields

File: scan/classifier_test.go [hunk 2]
```diff
@@ -131,10 +131,10 @@ func TestClassifierInclude(t *testing.T) {
...
+			{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification/transitive/mods"},
```
Comment (golangci) [inline scan/classifier_test.go]: github.com/go-swagger/go-swagger/scan.packageFilter composite literal uses unkeyed fields

File: scan/classifier_test.go [hunk 3]
```diff
@@ -131,10 +131,10 @@ func TestClassifierInclude(t *testing.T) {
...
+			{"github.com/go-swagger/go-swagger/fixtures/goparsing/classification/operations"},
```
Comment (golangci) [inline scan/classifier_test.go]: github.com/go-swagger/go-swagger/scan.packageFilter composite literal uses unkeyed fields

**Comment by fredbi** [comment]
Trying here to reduce the cost of future changes

**Comment by JonKohler** [comment]
Can we override this PR to push it in (just the codegov issue looks outstanding)?

**Comment by fredbi** [comment]
Yes I know. I need an approval by one of the repo admins

**Comment by fredbi** [comment]
@jonkohler today i pushed golangci yaml and badge to almost all our repos.
Doing so I ran gometalinter + fixed all issues reported by goreport card

**Comment by JonKohler** [comment]
fantastic! Thats greally great. Did you also run the golangci-lint tool they provide? Thats the "on-prem" tool from their hosted service, so you can see exactly what they are going to find. 

Tends to be a heck of a lot faster IMHO than gometalinter for whatever reason

**Comment by fredbi** [comment]
No I did not know this tool.
Will try it.

**Comment by fredbi** [comment]
I tried the "on-prem" tool on go-openapi/analysis (started with a small repo). Pfew... That's a lot of work to put everything to the level in one pass! But well, it is lowering our technical debt. So thanks for the tip.

File: scan/scanner.go [hunk 4]
```diff
@@ -103,7 +103,6 @@ var (
 
 	rxIn              = regexp.MustCompile(`[Ii]n\p{Zs}*:\p{Zs}*(query|path|header|body|formData)$`)
 	rxRequired        = regexp.MustCompile(`[Rr]equired\p{Zs}*:\p{Zs}*(true|false)$`)
-	rxExample         = regexp.MustCompile(`[Ex]ample\p{Zs}*:\p{Zs}*(.*)$`)
```
[the remark this answers is not in the document]
Comment (fredbi) [reply scan/scanner.go]: It is below, because it was unused (I just commented it out)

**Pull request merged**